core/src/main/java/tc/oc/pgm/scoreboard/SidebarMatchModule.java

```diff
@@ -90,9 +90,15 @@ public SidebarMatchModule createMatchModule(Match match) throws ModuleLoadExcept
 
   protected @Nullable Future<?> renderTask;
 
-  private static String renderSidebarTitle(Collection<MapTag> tags) {
+  private static String renderSidebarTitle(
+      Collection<MapTag> tags, @Nullable Component gamemodeName) {
     final Component configTitle = PGM.get().getConfiguration().getMatchHeader();
-    if (configTitle != null) return LegacyComponentSerializer.legacy().serialize(configTitle);
+    if (configTitle != null)
+      return LegacyComponentSerializer.legacy().serialize(configTitle.color(TextColor.AQUA));
```

Comment:
Do we need to use `TextTranslations` here? This won't translate the gamemodes, as-is.

Reply:
I believe that’s for the config scoreboard sidebar header. Also since scoreboards are for all, we can’t translate them yet correct?